cmd/otelcol/config/collector/agent_config.yaml

```diff
@@ -66,6 +66,9 @@ receivers:
             regex: '.*grpc_io.*'
             action: drop
   sapm:
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+    listenAddress: 127.0.0.1:9080
```

Comment:
```suggestion
    listenAddress: 0.0.0.0:9080
```

Reply:
Clearly debatable, but I believe all the other receivers above default to 0.0.0.0 -- if so we should do the same thing here